src/character.cpp

```diff
@@ -1710,6 +1710,11 @@ bool Character::is_running() const
     return move_mode->type() == move_mode_type::RUNNING;
 }
 
+bool Character::is_jogging() const
+{
+    return move_mode->type() == move_mode_type::JOGGING;
```

Comment:
```suggestion
    return move_mode == move_mode_id( "jogging" );
```

Reply:
Should I also change Character::is_running?

Reply:
I missed this, and it's some irrelevant now, but I think renaming this whole trio is_running/walking/crouching and the enum associated might be a good idea.